src/overlays/actors/ovl_En_Mm3/z_en_mm3.c

```diff
@@ -63,15 +62,27 @@ static ColliderCylinderInit sCylinderInit = {
 
 static CollisionCheckInfoInit2 sColChkInfoInit = { 0, 0, 0, 0, MASS_IMMOVABLE };
 
-static AnimationInfo sAnimationInfo[] = {
-    { &object_mm_Anim_002238, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -7.0f },
-    { &object_mm_Anim_00A4E0, -1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -7.0f },
-    { &object_mm_Anim_00C640, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -7.0f },
-    { &object_mm_Anim_00A4E0, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -7.0f },
-    { &object_mm_Anim_000468, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -7.0f },
-    { &object_mm_Anim_00CD90, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -12.0f },
-    { &object_mm_Anim_00DA50, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -12.0f },
-    { &object_mm_Anim_00DA50, 1.0f, 0.0f, 10.0f, ANIMMODE_ONCE, -10.0f },
+typedef enum EnMm3Animation {
+    /* 0 */ ENMM3_ANIM_0,
+    /* 1 */ ENMM3_ANIM_1,
+    /* 2 */ ENMM3_ANIM_2,
+    /* 3 */ ENMM3_ANIM_3,
+    /* 4 */ ENMM3_ANIM_4,
+    /* 5 */ ENMM3_ANIM_5,
+    /* 6 */ ENMM3_ANIM_6,
+    /* 7 */ ENMM3_ANIM_7,
+    /* 8 */ ENMM3_ANIM_MAX
+} EnMm3Animation;
+
+static AnimationInfo sAnimationInfo[ENMM3_ANIM_MAX] = {
+    { &object_mm_Anim_002238, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -7.0f },   // ENMM3_ANIM_0
+    { &object_mm_Anim_00A4E0, -1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -7.0f },  // ENMM3_ANIM_1
+    { &object_mm_Anim_00C640, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -7.0f },   // ENMM3_ANIM_2
+    { &object_mm_Anim_00A4E0, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -7.0f },   // ENMM3_ANIM_3
+    { &object_mm_Anim_000468, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -7.0f },   // ENMM3_ANIM_4
+    { &object_mm_Anim_00CD90, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -12.0f },  // ENMM3_ANIM_5
+    { &object_mm_Anim_00DA50, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -12.0f },  // ENMM3_ANIM_6
+    { &object_mm_Anim_00DA50, 1.0f, 0.0f, 10.0f, ANIMMODE_ONCE, -10.0f }, // ENMM3_ANIM_7
```

Comment:
er, isn't this going to clash with the postman pr?

Reply:
Yeah, good point. That PR just focuses on the object, so this still has new parts. But it's also done by a new contributor. I can help guide them merge the conflict if need be, I'll keep a close eye on that PR